gtfs-realtime/spec/en/reference.md

```diff
@@ -73,6 +73,13 @@ Fields labeled as **experimental** are subject to change and not yet formally ad
 
 The contents of a feed message. Each message in the stream is obtained as a response to an appropriate HTTP GET request. A realtime feed is always defined with relation to an existing GTFS feed. All the entity ids are resolved with respect to the GTFS feed.
 
+A feed depends on these external configurations:
```

Comment:
I don't think we need this paragraph - it doesn't add much value.

Reply:
I'm removing it this evening :)